### ISSUE TITLE
problem with users.sql file

### ISSUE THREAD

**Issue by priyankashinolikar** [pranavkhurana/Time-table-scheduler#7]
unknown character set 'utf8mb4' gives error while importing sql file kindly give any solution for the same

**Comment by noahire** [comment]
Hello, please how were you able to solve the "out of bound exception error"
Am still having problem with it..

**Comment by varnit02** [comment]
@noahire yes  i am also getting it, @priyankashinolikar how have you solved it
